journalapp/test/test_views.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""Testing views by calling the views functions directly."""
+
+from journalapp.views import (
+    list_view,
+    detail_view,
+    add_entry,
+    edit_entry,
+)
+
+from journalapp.forms import EditEntryForm, AddEntryForm
+
+
+def test_list_view(dbtransaction, new_entry, dummy_get_request):
+    """Test that list_view returns a Query of Entries."""
+    response_dict = list_view(dummy_get_request)
+    entries = response_dict['entries']
+    assert entries[0] == new_entry
+
+
+def test_detail_view(dbtransaction, new_entry, dummy_get_request):
+    """Test that detail_view returns the correct Entry page."""
+    dummy_get_request.matchdict = {'entry_id': new_entry.id}
+    response_dict = detail_view(dummy_get_request)
+    assert response_dict['entry'] == new_entry
```

Comment:
You have a view below that tests 404 for the edit view.  You should have a similar test for the detail view.